api/v1beta1/keystoneendpoint.go

```diff
@@ -190,3 +190,42 @@ func DeleteKeystoneEndpointWithName(
 
 	return nil
 }
+
+// GetKeystoneEndpointList func
+func GetKeystoneEndpointList(
+	ctx context.Context,
+	h *helper.Helper,
+	namespace string,
+) (*KeystoneEndpointList, error) {
+
+	ke := &KeystoneEndpointList{}
+	err := h.GetClient().List(ctx, ke)
+	if err != nil {
+		return nil, err
+	}
+
+	return ke, nil
+}
+
+// GetKeystoneEndpointUrls returns all URLs currently registered
```

Comment:
can you please add a description of what visibility is  used for, e.g. like the `--interface` from the openstackclient, could also rename interface to call it like the openstackclient.

```
  --interface <interface>
                        Filter by interface type (admin, public or internal)
```

 also do we want to do this e.g. a string ptr and if it is nil, we return all of them and not filter on visibility?